go.mod

```diff
@@ -1,89 +1,80 @@
 module github.com/kedacore/keda-olm-operator
 
-go 1.18
+go 1.19
 
 require (
-	github.com/go-logr/logr v1.2.3
+	github.com/go-logr/logr v1.2.4
 	github.com/manifestival/controller-runtime-client v0.4.0
-	github.com/manifestival/manifestival v0.7.1
+	github.com/manifestival/manifestival v0.7.2
 	github.com/onsi/ginkgo v1.16.5
-	github.com/onsi/gomega v1.24.1
+	github.com/onsi/gomega v1.27.6
 	github.com/openshift/api v3.9.0+incompatible
-	k8s.io/api v0.25.4
-	k8s.io/apimachinery v0.25.4
-	k8s.io/apiserver v0.25.0
-	k8s.io/client-go v0.25.4
-	k8s.io/kube-aggregator v0.24.3
-	sigs.k8s.io/controller-runtime v0.13.1
+	k8s.io/api v0.26.3
```

Comment:
I think we can bump k8s deps to 0.26.4 - this version wasn't present when we made 2.10.1 upstream.

Reply:
Thanks, done